plugin-gradle/src/main/java/com/diffplug/gradle/spotless/KotlinGradleExtension.java

```diff
@@ -102,6 +102,16 @@ public void dropboxStyle() {
 			replaceStep(createStep());
 		}
 
+		public void googleStyle() {
+			style = Style.GOOGLE;
+			replaceStep(createStep());
+		}
+
+		public void kotlinlangStyle() {
+			style = Style.KOTLINLANG;
+			replaceStep(createStep());
```

Comment:
Seems good to refactor KotlinGradleExtension to have the same `void style(Style style)` method above.

Reply:
Done :)

Reply:
Half-done :) 

Current

```java
		public void googleStyle() {
			style = Style.GOOGLE;
			replaceStep(createStep());
		}
```

Better

```java
		public void googleStyle() {
			style(Style.GOOGLE);
		}
```